### DISCUSSION TITLE
Combining Statistics

### DISCUSSION THREAD

**Discussion by M0TZO** [magicbug/Cloudlog#971] [General]
As someone who loves stats, I was wondering if would be possible to have a way to combine different logs together for stats.

For myself so I could add when I've operated M0TZO abroad (_ok was only once, and one call worked_) and I keep my Digital Voice (DMR, D-Star, YSF, etc) in a separate log.

I see this as an option that could be available in the Edit button on Station Locations.

Would also be useful if people operating SOTA, etc at different Gridsquares (eg (_JO01_) and a log for each new Square (_I know of at least one person who has/is doing this_).

Hope this makes sense, and if so Thanks in advance.
M0TZO

**Comment by magicbug** [comment]
It's coming you'll find this is a duplicate request, once logbooks are some with station location linking.

**Reply by M0TZO** [reply]
Excellent news.

**Reply by myzinsky** [reply]
Hey, that would be really super duper awesome :-) Any news when it will happen?

**Reply by myzinsky** [reply]
@magicbug Hey Peter, any update on this?

**Reply by magicbug** [reply]
If you dont mind risking the dev branch see "station_logbook" else follow along on #1131 its mostly there.
